workflow/rules/annotation/pass2.smk

```diff
@@ -44,7 +44,7 @@ rule annotate_dgidb:
     resources:
         dgidb_requests=1
     shell:
-        "rbt vcf-annotate-dgidb {input} > {output}"
+        "rbt vcf-annotate-dgidb {input} | bcftools view -Ob > {output}"
```

Comment:
Sorry, just seen this. I'd rather have rbt print BCF directly. This is how all other rbt commands should be implemented already. Can you change that accordingly?

Reply:
Sure, I will check on that.